explore/src/main/scala/explore/proposal/ProgramUsersTable.scala

```diff
@@ -138,8 +193,8 @@ object ProgramUsersTable:
       .withHooks[Props]
       .useContext(AppContext.ctx)
       .useStateView(IsActive(false))
-      .useMemoBy((_, _, _) => ()): (_, ctx, _) => // cols
-        _ => columns(ctx)
+      .useMemoBy((_, _, _) => ()): (p, ctx, _) => // cols
```

Comment:
Can `users` change externally? Eg: editing in another session.

If so, it should go in a `TableMeta`.